server/routers/share-recovery/registerAuthShare.ts

```diff
@@ -70,8 +70,6 @@ export const registerAuthShare = protectedProcedure
     // TODO: Add a wallet activation attempt limit?
 
     if (challengeErrorMessage) {
-      console.error(challengeErrorMessage);
```

Comment:
Because these are now logged automatically, I removed all the ones that we were logging manually (except for those logs that provide additional details on top of the error message, like those in the challenge validation logic).